ansible/roles/baselayout/vars/main.yml

```diff
@@ -75,6 +75,10 @@ packages: {
     'gcc-10,g++-10,ccache,git,curl,libfontconfig1,apt-transport-https,ca-certificates,sudo,python3-pip',
   ],
 
+  debian12: [
+    'gcc-12,g++-12,ccache,git,curl,libfontconfig1,apt-transport-https,ca-certificates,sudo,python3-pip,python3-virtualenv',
```

Comment:
Can we use Python's built-in [`venv`](https://docs.python.org/3/library/venv.html) via `python3-venv` and `python3-full` instead?

Reply:
Sure, let me try.

Reply:
What's the command to run in this case? Ansible defaults to `virtualenv` which doesn't exist with `python3-venv`: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html#parameter-virtualenv_command

Reply:
I went with `virtualenv_command: python3 -m venv`

Reply:
Install the venv: `python3 -m venv name_of_my_venv`
Activate the venv (optional): `source name_of_my_venv/bin/activate`